json-guides/microprofile-config.json

```diff
@@ -15,6 +15,35 @@
             ]
         },
         {
+            "name": "ConfigurePropsFile",
+            "title": "Configure using a Properties File",
+            "description": [
+                "In Websphere Liberty, you can provide a properties file as part of your Liberty environment configuration. This properties file lets you specify configuration for overriding injected values.",
```

Comment:
The properties file could be provided in different way. The easy way is to package it as part of the application. If we want to provide it as part of the liberty environment, it has to be packaged as a shared lib.